src/helpers/__tests__/helpers.test.ts

```diff
@@ -3,132 +3,235 @@
+    const timestamp = moment().subtract(30, 'seconds').toISOString();
+    const timeAgo = getTimeDisplayHowLongAgo(timestamp);
+    expect(timeAgo).toBe('a few seconds ago');
+  });
 
-it('should return "a minute ago" for a timestamp that is exactly one minute old', () => {
-  const timestamp = moment().subtract(1, 'minute').toISOString();
-  const timeAgo = getTimeDisplayHowLongAgo(timestamp);
-  expect(timeAgo).toBe('a minute ago');
-});
+  it('should return "a minute ago" for a timestamp that is exactly one minute old', () => {
+    const timestamp = moment().subtract(1, 'minute').toISOString();
+    const timeAgo = getTimeDisplayHowLongAgo(timestamp);
+    expect(timeAgo).toBe('a minute ago');
+  });
 
-it('should return "a month ago" for a timestamp that is more than a month old', () => {
-  const timestamp = moment().subtract(1, 'month').subtract(1, 'day').toISOString();
-  const timeAgo = getTimeDisplayHowLongAgo(timestamp);
-  expect(timeAgo).toBe('a month ago');
-});
+  it('should return "a month ago" for a timestamp that is more than a month old', () => {
+    const timestamp = moment().subtract(1, 'month').subtract(1, 'day').toISOString();
+    const timeAgo = getTimeDisplayHowLongAgo(timestamp);
+    expect(timeAgo).toBe('a month ago');
+  });
+
+  it('should return "a day ago" for a timestamp that is more than 24 hours old', () => {
+    const timestamp = moment().subtract(25, 'hours').toISOString();
+    const timeAgo = getTimeDisplayHowLongAgo(timestamp);
+    expect(timeAgo).toBe('a day ago');
+  });
 
-it('should return "a day ago" for a timestamp that is more than 24 hours old', () => {
-  const timestamp = moment().subtract(25, 'hours').toISOString();
-  const timeAgo = getTimeDisplayHowLongAgo(timestamp);
-  expect(timeAgo).toBe('a day ago');
+  it('should throw an error if the given timestamp is not a valid date', () => {
+    const timestamp = 'invalid';
+    expect(() => {
+      getTimeDisplayHowLongAgo(timestamp);
+    }).toThrow('Invalid timestamp');
+  });
 });
 
-it('should throw an error if the given timestamp is not a valid date', () => {
-  const timestamp = 'invalid';
-  expect(() => {
-    getTimeDisplayHowLongAgo(timestamp);
-  }).toThrow('Invalid timestamp');
+describe('getAuthType', () => {
+  it('should return AuthType.UsernameAndPassword when credential has username and password', () => {
+    const mockCredential: CredentialType = {
+      id: 'mockId',
+      name: 'mockName',
+      created_at: new Date(),
+      updated_at: new Date(),
+      cred_type: 'mockCredType',
+      username: 'testUser',
+      password: 'testPassword',
+      ssh_keyfile: '',
+      auth_token: '',
+      ssh_passphrase: '',
+      become_method: '',
+      become_user: '',
+      become_password: '',
+      sources: []
+    };
+    const result = getAuthType(mockCredential);
+    expect(result).toBe(authType.UsernameAndPassword);
+  });
```

Comment:
required:
move to using a loop/array/list to simplify this test, we're repeating the same function call with slightly different data

future:
Consider moving the helper `getAuthType` to a support file associated with the `viewCredentialsList.tsx` file like `viewCredentialsListHelpers.ts` or `credentialsHelpers.ts`... see future below

future:
We'll need to review string enums and consider getting rid of the entire function `getAuthType` and opting for generating a `i18n` string key from `Credential` properties